frontend/src/admin/common/tables/SortedTable.js

```diff
@@ -1,6 +1,16 @@
 import React, { Component } from 'react';
+import styled from 'styled-components';
+
 import Table from './Table';
-import sortColumn from '../../lib/sortColumn';
+import sortColumn from '../../../lib/sortColumn';
```

Comment:
Yeah the whole lib folder is a hangover and most of the files and shared any more. I'll finish moving anything left out of it.